README.md

```diff
@@ -71,7 +71,7 @@ For example, if you want to provide a different path to your configuration, you
     const config = require('openshift-rest-client').config;
 
     const path '~/some/path/config';
-    const customConfig = config.fromKubeconfig(path);
+    const customConfig = config.loadFromFile(path);
```

Comment:
Unfortunately this is the only breaking change for current users.
IMO there's no other way since [getting a config like this](https://github.com/nodeshift/openshift-rest-client/pull/171/files#diff-168726dbe96b3ce427e7fedce31bb0bcL23) is [deprecated](https://github.com/godaddy/kubernetes-client/blob/master/merging-with-kubernetes.md#requirekubernetes-clientconfig)